moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -0,0 +1,303 @@
+        }
+        return self
+
+    def joint_limits(self, file_path: Optional[str] = None):
+        self.__moveit_configs.joint_limits = {
+            self.__robot_description
+            + "_planning": load_yaml(
+                self._package_path / (file_path or "config/joint_limits.yaml")
+            )
+        }
+        return self
+
+    def moveit_cpp(self, file_path: Optional[str] = None):
+        self.__moveit_configs.moveit_cpp = load_yaml(
+            self._package_path / (file_path or "config/moveit_cpp.yaml")
+        )
+        return self
+
+    def trajectory_execution(
+        self,
+        file_path: Optional[str] = None,
+        moveit_manage_controllers: bool = True,
+    ):
+        self.__moveit_configs.trajectory_execution = {
+            "moveit_manage_controllers": moveit_manage_controllers,
+        }
+        self.__moveit_configs.trajectory_execution.update(
+            load_yaml(
+                self._package_path
+                / (file_path or f"config/{self.__robot_name}_controllers.yaml")
+            )
+        )
+        return self
+
+    def planning_scene_monitor(
+        self,
+        publish_planning_scene: bool = True,
+        publish_geometry_updates: bool = True,
+        publish_state_updates: bool = True,
+        publish_transforms_updates: bool = True,
+    ):
+        self.__moveit_configs.planning_scene_monitor = {
+            # TODO: Fix parameter namespace upstream -- see planning_scene_monitor.cpp:262
+            # "planning_scene_monitor": {
+            "publish_planning_scene": publish_planning_scene,
+            "publish_geometry_updates": publish_geometry_updates,
+            "publish_state_updates": publish_state_updates,
+            "publish_transforms_updates": publish_transforms_updates,
+            # }
+        }
+        return self
+
+    def planning_pipelines(
+        self, default_planning_pipeline: str = None, pipelines: List[str] = None
+    ):
+        if pipelines is None:
+            pipelines = ["ompl"]
+            default_planning_pipeline = pipelines[0]
+        if default_planning_pipeline not in pipelines:
+            raise RuntimeError(
+                f"default_planning_pipeline: `{default_planning_pipeline}` doesn't name any of the input pipelines "
+                f"`{','.join(pipelines)}`"
+            )
+        self.__moveit_configs.planning_pipelines = {
+            "planning_pipelines": pipelines,
+            "default_planning_pipeline": default_planning_pipeline,
+        }
+        for pipeline in pipelines:
+            self.__moveit_configs.planning_pipelines[pipeline] = load_yaml(
+                self._package_path / "config" / (pipeline + "_planning.yaml")
+            )
+        return self
+
+    def cartesian_limits(self, file_path: Optional[str] = None):
+        self.__moveit_configs.cartesian_limits = load_yaml(
+            self._package_path / (file_path or "config/cartesian_limits.yaml")
+        )
```

Comment:
Thanks for adding the cartesian limits! Similar to the joint limits, cartesian limits where within the robot_description_planning namespace i.e. in the [panda_moveit_config](https://github.com/ros-planning/moveit_resources/blob/master/panda_moveit_config/launch/planning_context.launch). I guess it would make sense to keep it that way for the sake of uniformity. What do you think?
```suggestion
        self.__moveit_configs.cartesian_limits = {
            self.__robot_description
            + "_planning": load_yaml(
                self._package_path / (file_path or "config/cartesian_limits.yaml")
            )
        }
```